torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1859,6 +1859,26 @@ def sample_inputs_masked_select(op_info, device, dtype, requires_grad):
            decorators=[_wrap_maybe_warns("floor_divide is deprecated, and will be removed")],
            supports_autograd=False,
            ),
+    UnaryUfuncInfo('frexp',
+                   op=torch.frexp,
+                   ref=np.frexp,
+                   dtypesIfCPU=floating_types_and(torch.half),
+                   dtypesIfCUDA=floating_types_and(torch.half),
+                   # skip testing torch.frexp as it is not supported by ROCm platform yet
+                   decorators=[skipCUDAIfRocm],
+                   test_inplace_grad=False,
+                   skips=(
+                       # skip OpInfo `test_reference_numerics_*` and `test_out_arg_all_dtypes`.
+                       # torch.frexp returns two tensors, a floating mantissa and a int32 exponent.
+                       # It is hard to update the above OpInfo tests to support testing two output tensors
+                       # with different dtype.
+                       # Test out-variants as well as extremal values with torch.frexp
+                       # in the specific test methods `test_frexp_reference_numerics` and `test_frexp_out`.
+                       SkipInfo('TestUnaryUfuncs', 'test_reference_numerics_normal'),
+                       SkipInfo('TestUnaryUfuncs', 'test_reference_numerics_hard'),
+                       SkipInfo('TestUnaryUfuncs', 'test_reference_numerics_extremal'),
+                       SkipInfo('TestUnaryUfuncs', 'test_out_arg_all_dtypes'),
```

Comment:
Rather then skipping this test, could you fix the test to query whether the op supports out tensor as in https://github.com/pytorch/pytorch/blob/baed2cfe0110610274f64d217af587a4a5c8dedd/test/test_ops.py#L343.

Reply:
Addressed.